Makefile

```
@@ -117,6 +118,9 @@ TMT_TEST_IMAGES_TARGETS := $(foreach image,$(TMT_TEST_CONTAINER_IMAGES),images-t
 images-tests: $(TMT_TEST_IMAGES_TARGETS)  ## Build customized images for tests
 	podman images | grep 'localhost/$(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/' | sort
 
+images-tests-bases:  ## Download base images for custom test images
+	for image in $(TMT_TEST_IMAGE_BASES); do podman pull "$$image"; done
```

pull supports multiple images

```suggestion
	podman pull $(TMT_TEST_IMAGE_BASES)
```

---

Nice, addressed in d5c933e4.